### PR TITLE
Sign up form

### PR DESCRIPTION
**Pull request by GraceOyiza**
Waiting for approval to merge.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            <p class="mt-5 x-small-text">
+              Standard call, messaging or data rates may apply
+            </p>
+          </div>
+          <div class="vertical-padding-16">
+            <label for="">Password</label>
+            <input class="mt-10" type="text" id="" name="" />
+            <i class="fa fa-unlock input-icon"></i>
+          </div>
+          <button id="submit-btn" class="larger-text">
```
Comment (HADDADSOHAIB) [inline index.html]: PLease fix the type of the input and buttons:
- [x] It has an `<input>` of type `email` and a label `Email address` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
- [x] It has an `<input>` of type `tel` and a label `Phone` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
- [x] It has an `<input>` of type `password` and a label `Password` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
- [x] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello everyone    :handshake: 

### Good job so far :clap:  your project looks very good and similarto the original :medal_sports: , you did great job on the design.

### Before we could approve the project, there some mandatory changes:
- Please fix the validator issues:
Screenshot from 2020-06-12 11-10-00
- Check my comments for additional requirements.
- I see three participants in this repository, only two people can work in a collaborative project, please provide an explanation for that.
### Once you are done with requirements, ask for another code review.

**Comment by El-Potato-Slayer** [comment]
>     * I see three participants in this repository, only two people can work in a collaborative project, please provide an explanation for that.
There are two participants. The project owner had someone else complete the trails for microverse on her computer, and forgot to change back to her details.

**Comment by santiago-dev96** [comment]
# PR under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — APPROVED** [review]
# Project approved! :tada:

Good job! You fulfill all the requirements for approval, you may continue with the next projects, good luck! 👍

## P.D.

Better remove stickler CI from this repo.

**Pull request merged**